src/Alarm.cpp

```diff
@@ -82,44 +120,71 @@ void Alarm::arm(){
    _isArmed = true;
  }
 }
-
+/**
+  @return returns true if the isArmed is not armed and isCalibrated is
+    calibrated, and isTripped and isTriggered are not activated
+*/
 bool Alarm::isReadyToArm(){
   return(!this->isArmed()
         && this->isCalibrated()
         && !this->isTripped()
         && !this->isTriggered());
 }
-
+/**
+  Getter for the param _isArmed
+*/
 bool Alarm::isArmed(){
   return _isArmed;
 }
-
+/**
+  @return returns a true false based on whether or the reading taken is less
+    than the _baseReading by 100
+*/
 bool Alarm::isTripped(){
   return (_photoR->takeReading() - 100 < _baseReading);
 }
-
+/**
+  Sets the alarm to be triggered
+  @param sets _isTriggered to be true
+  @param sets _isSilenced to be false
+*/
 void Alarm::trigger(){
   Logger::Log("Alarm has been triggered!");
   _isTriggered = true;
   _isSilenced = false;
 }
-
+/**
+  If _isSilenced is not set soundAlarmHighTone and soundAlarmLowTone are used
+    to set tone high and low, also tells _redLED and _alarmLED to flash
+*/
```

Comment:
The reason for this method is that I ran into an issue in the main class where if the alarm was triggered it wouldn't listen to any other commands sent to it. If you notice in the loop now it still checks for things like "Is the button pressed" etc, but sounds this single alarm cycle every loop. This allows us to still send commands to the Arduino even when it's triggered.